### PR TITLE
Nuke translations

### PR DESCRIPTION
**Pull request by leemannd**
A lot of time the overwrite parameter is not working.
Nuking them before using the reload works well

### CODE REVIEW

**Review by sebalix — APPROVED** [review]
:+1: pre-commit is complaining but pre-approving

File: anthem/lyrics/modules.py [hunk 1]
```diff
@@ -41,3 +42,12 @@ def update_translations(ctx, module_list, overwrite=False):
     ctx.log_line("Reloading translations for %s" % str(module_list))
     mods.with_context(overwrite=overwrite)
     getattr(mods, method_name)()
+
+
+def nuke_translations(ctx, module_list):
+    """ Remove translations from module list"""
+    if not isinstance(module_list, list):
+        raise AnthemError(
+            u"You have to provide a list of " "module's name to remove the translations"
```
Comment (mmequignon) [inline anthem/lyrics/modules.py:51]: second string should be on the second line, or merged in a single string.

**Review by mmequignon — APPROVED** [review]
pre-approving as well

File: anthem/lyrics/modules.py (hunk 1 above)
Comment (leemannd) [reply anthem/lyrics/modules.py:51]: Merci

**Pull request merged**